docs/labs/index.rst

```diff
@@ -15,17 +15,6 @@ If we get lots of positive feedback about any one of these projects then we will
 
 .. _labs-demo:
 
-Flocker Docker plugin with CLI and GUI
```

Comment:
Please revert deleting this section. We can link to the official plugin docs, but this is a really important demo and it remains valid.